pkg/apis/camel/v1alpha1/build_types.go

```diff
@@ -22,6 +22,7 @@ type BuildSpec struct {
 	Dependencies   []string                `json:"dependencies,omitempty"`
 	Repositories   []string                `json:"repositories,omitempty"`
 	BuildDir       string                  `json:"buildDir,omitempty"`
+	BuilderImage   string                  `json:"builderImage,omitempty"`
```

Comment:
Ideally, I would avoid exposing a field in public CRDs, solely for testing purpose. There may be a way to retrieve the image form the operator used to run it, and use that image for the other resources.

Reply:
That can be a good alternative... thinking about it...

Reply:
We can probably inject it through downward apis

Reply:
Exactly. The image field may not be directly accessible via the Downward API, thought the pod info are and can be used to get the pod spec.